src/ConfigHandler.js

```diff
@@ -92,6 +92,8 @@ export default class {
             }
         } catch (e) {
             console.log(e);
+            // give user better understanding what is going wrong with the YAML config
+            alert(e);
```

Comment:
I recommend to return the errocode to caller and print error there. I would not recommend UI interactions at confighandler